tidy3d/web/api/container.py

```diff
@@ -176,25 +199,23 @@ def run(self, path: str = DEFAULT_DATA_PATH) -> SimulationDataType:
             Object containing simulation results.
         """
 
+        self.upload()
         self.start()
         self.monitor()
         return self.load(path=path)
```

Comment:
https://github.com/flexcompute/tidy3d/pull/1647
see the PR above about my comments. 
we could start the download early once the job is done without waiting for all tasks get done. 
the same ideas to upload the tasks as well.  we don't need wait for all files get upload then start run. 
1. the benefit is to reduce the peak concurrency of batch jobs.
2. and reduce the overall time for the whole batch.